src/test/suite/recorded.test.ts

```diff
@@ -147,35 +147,29 @@ async function runTest(file: string) {
 
   disposeFakeIde();
 
-  if (fixture.thrownError != null) {
-    throw Error(
-      `Expected error ${fixture.thrownError.name} but none was thrown`
-    );
-  }
-
```

Comment:
Needed to move this so that we could update fixtures that went from error to success